eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig

```diff
@@ -10,12 +10,14 @@
     {% set defaultValue = fielddefinition.defaultValue.text %}
     {{ block( 'settings_defaultvalue' ) }}
     <li class="max-length">
-        <span>Max string length:</span>
+        <div class="fielddef-settings-name">Max string length:</div>
```

Comment:
since this template was not used anywhere before, it's a good opportunity to adopt the `ez-` prefix on the classes here as well and to avoid the shortcut names so ez-fielddefintion-setting-name, ez-fielddefinition-setting-value, ... (setting not settings by the way)